tools/generate_schema_wrapper.py

```diff
@@ -193,16 +160,19 @@ class DatumChannelMixin:
     def to_dict(
         self,
         validate: bool = True,
-        ignore: Optional[List[str]] = None,
-        context: Optional[TypingDict[str, Any]] = None,
+        ignore: list[str] | None = None,
+        context: dict[str, Any] | None = None,
     ) -> dict:
         context = context or {}
         ignore = ignore or []
-        datum = self._get("datum", Undefined)  # type: ignore[attr-defined]
+        datum = self._get("datum", Undefined)  # type: ignore[attr-defined] # noqa
         copy = self  # don't copy unless we need to
-        if datum is not Undefined:
-            if isinstance(datum, core.SchemaBase):
-                pass
+        # NOTE: The block below does nothing?
```

Comment:
This is a PR comment and should not be part of the code.

And fully agree, that code can be removed.